scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -30,6 +30,25 @@ class BignumModRawTarget(test_data_generation.BaseTarget):
 
 # BEGIN MERGE SLOT 2
 
+class BignumModRawSub(bignum_common.ModOperationCommon,
+                      BignumModRawTarget):
+    """Test cases for bignum mpi_mod_raw_sub()."""
+    symbol = "-"
+    test_function = "mpi_mod_raw_sub"
+    test_name = "mbedtls_mpi_mod_raw_sub"
+    input_style = "fixed"
+    arity = 2
+
+    def arguments(self) -> List[str]:
+        return [bignum_common.quote_str(n) for n in [self.arg_a,
+                                                     self.arg_b,
+                                                     self.arg_n]
+               ] + self.result()
```

Comment:
@minosgalanakis has concerns about doing this (raised on my PR at https://github.com/Mbed-TLS/mbedtls/pull/6656#discussion_r1031756658 )- I shamelessly copied this; it seems fine to me - opinions from others?

Reply:
Updating: for consistency it might be better to remove this override and just reorder the `input_FOO` variables in the test function (which is what I did in #6656)

Reply:
I think we should adopt the parameter order that Gilles suggested, but not in these two PRs. I am going to raise a new PR with a fix once these two are in.